qutebrowser/config/configdata.yml

```diff
@@ -1042,6 +1042,44 @@ editor.encoding:
   default: utf-8
   desc: Encoding to use for the editor.
 
+## fileselect
+
+fileselect.handler:
+  default: default
+  type:
+    name: String
+    valid_values:
+      - default: "Use the default file selector."
+      - external: "Use an external command."
```

Comment:
Could you mention the other two settings (`fileselect.single_file.command` and `fileselect.multiple_files.command`) in this setting, to point people to configure those?